### ISSUE TITLE
Towards a more complete Analysis.Export

### ISSUE THREAD

**Issue by chkl** [visq/language-c#50]
Hi everyone,

I was already talking a bit with lambdageek on reddit (InformalInflation, that's me). In short, I have this use case where I would like to apply some tranformations to a C file *after* type checking. (I need the type information to perform the transformations). As lambdageek pointed out, I should probably do this on the `SemRep` and not on the AST. 

So, after applying my transformation on `SemRep` I still need to pretty-print it back into the file. I believe that is what `Analysis.Export` is for. 

I saw the warning:
```
WARNING : This is just an implementation sketch and not very well tested.
```

And well, if I am not mistaken, there's no top-level function like `exportSemRep :: GlobalDecls -> CTranslUnit`, which would be what I need.

Before I start adding functions towards the goal, is there something I should know?

tl;dr: I would like to contribute to `Analysis.Export`, what do I need to know?
